src/libm-tester/iutsimdmain.c

```diff
@@ -0,0 +1,57 @@
+//          Copyright Naoki Shibata 2010 - 2017.
```

Comment:
I am sorry for that, but it would be too much overhead. I will have smaller amount of time I can spend for this project after June, so I think that would solve this problem.